lesson_11/arrays_java/arrays_app/src/main/java/com/codedifferently/lesson11/Lesson11.java

```diff
@@ -9,14 +10,32 @@ public class Lesson11 {
    * https://leetcode.com/problems/concatenation-of-array
    */
   public int[] getConcatenation(int[] nums) {
-    return null;
-  }
+    int n = nums.length;
+    int[] ans = new int[n*2];
+
+    for (int i=0; i<n; i++) {
+      ans[i]=nums[i];
+      ans[i+n]=nums[i];
+      } return ans;
+    } 
+  
 
   /**
    * Provide the solution to LeetCode 2942 here:
    * https://leetcode.com/problems/find-words-containing-character/
    */
   public List<Integer> findWordsContaining(String[] words, char x) {
-    return null;
+    var words = new ArrayList<Integer>();
+    int wordArray = (words.length)
+    // int newValue = (originalArray + 1);
+
+    // int[] newArray = new int[words.length + 1];
+    
+   for (int i=0; i<originalArray; i++) {
+     if (words[i].contains(Character.toString(x)));
+     // System.arraycopy(words,i,newArray,i,words.length);
+      ArrayList.add(i);
```

Comment:
ArrayList is a class, you want to make a new ArrayList instance and then call add on that object.

```java
var results = new ArrayList<Integer>();
results.add(5);
```